CONTRIBUTING.md

```diff
@@ -74,6 +74,32 @@ environment variable to disable webpack optimizations.
 env DEBUG=true yarn start:prod
 ```
 
+## Authentication in your local Force app
+
+Authentication in Force is handled through OAuth, with [Gravity](https://github.com/artsy/gravity) authenticating the user and redirecting back to Force. For security reasons, the `localhost` origin [is forbidden as a redirect URL by Gravity in the staging environment](https://github.com/artsy/gravity/blob/543373d7d4413f5c8b1c8f84f73b2a592c00cba2/app/models/util/url_validation.rb#L23). This means that when running Force locally at `http://localhost:5000`, you won't be able to sign up or log in.
```

Comment:
This actually doesn't match my understanding. I _thought_ Force passed credentials into a purpose-built endpoint to get a valid access token, and _didn't_ rely on Oauth-style redirects. I'm not familiar with the security reasons you mention here either (but that doesn't mean you're wrong). Paging @mzikherman?

Reply:
"security reasons" is basically me guessing why we are only allowing `localhost` as a redirect origin except in `test` mode

Reply:
You're both sorta right! I think the text is fine as-is.

The Force <-> Gravity isn't the _standard_ oauth flow (as @joeyAghion points out). At the end, there is a custom step of navigating to `/user/sign_in` , initiating a Gravity session (in addition to the Force session just created), and redirecting back.

Generally, all the Gravity redirects (for session stuff, more proper oauth flow, etc.) don't allow redirects to external sites (so-called 'open redirect' vulnerability), so "security reasons" feels valid to keep.